### ISSUE TITLE
Design: Object page

### ISSUE THREAD

**Issue by darshi-shah** [saurabhd/hk_realestate#82]
Design for object page. 

This page should visually more attractive landing page for an overview on the object, including all apartments.

**Comment by darshi-shah** [comment]
Hi Rainer, 

Please see the design the this page. Please let us know which other details should be on this page. 

Thanks

![object](https://cloud.githubusercontent.com/assets/15724372/22139584/00e72690-df0d-11e6-8b8e-0d61a648ce3e.png)

**Comment by heliogabal** [comment]
thanks. It looks like this right now: https://herbert-kohlmeyer.de/objekt/stilvolle-stadtapartments-mitte-als-ideale-kapitalanlage 

The picture adds balance to this, but we were facing the following problem with the gallery on the apartment pages: Mostly, the facade pic that would be the one to show at the object page should be in portrait mode, which doesn't work well with the gallery we're having right now. So the idea was to only have one pics (the facade) on the object page, but in portrait mode. The other pics of the object are repeated in every apartment's gallery, so no real need to have a gallery here.

Then, it would also be advisable to show some more personality and give a sense of the human agents that stand behind H+K, so we could have a pic of the responsible agent with his contact details here as well.

At last, we can show the map to give a sense of location.

**Comment by darshi-shah** [comment]
Hi Rainer,

Please see revised mockup. 

Can you please explain about responsible agent role in terms of object page?  In CRM, responsible agents are based on inquiry. They are not related to apartment or object.

Thanks
![object](https://cloud.githubusercontent.com/assets/15724372/22281597/c5554ea0-e2fd-11e6-8b2f-2777cd8f10fb.jpg)

**Comment by heliogabal** [comment]
yes, there is a connection between an agent and the contacts he answers to, making him the responsible agent for that contact. Then, there is also the field "Ansprechpartner" for every object under "Objektverwaltung". These are not yet filled out for most objects, but provide the connection between an agent and an object, so he becomes the responsible agent for that object. 

If there is such a Ansprechpartner, you could show his profile pic and phone number together with some inviting text to contact him on that behalf.

**Comment by darshi-shah** [comment]
Hi Rainer,

Attached is revised design. It includes agent's details. Let  us know if we need to show any other detail regarding agent. 

Thanks

![object](https://cloud.githubusercontent.com/assets/15724372/22420404/48794f94-e708-11e6-8908-79b3cdade2a7.jpg)

**Comment by heliogabal** [comment]
yep, contact person is perfect. Instead of the map, we should have the object image, which in the future will likely be in portrait mode. Not sure if that is feasible, but you should give it a try. And the map after the apartment list.

**Comment by darshi-shah** [comment]
Since we are introducing full width images in apartment pages, should we use full width images on this page as well? Let us know please. 

Thanks

**Comment by heliogabal** [comment]
As the main image for the oject is likely to be a portrait mode in the future, that wouldn't work. So leave it as is, just replacing the map with a portrait mode pic. So we can verify it works that way.

**Comment by darshi-shah** [comment]
Hi Rainer,

Attached are the options for object page. Let us know your feedback for this.

![object1](https://cloud.githubusercontent.com/assets/15724372/25388360/54802c9c-29eb-11e7-81cc-8bc0c03d3e4e.jpg)
![object2](https://cloud.githubusercontent.com/assets/15724372/25388361/54d7e27a-29eb-11e7-8e00-503c73617fab.jpg)

Thanks

**Comment by heliogabal** [comment]
Thanks! As I said before, the new object photo is highly likely to be in portrait mode. Also, the landscape mode images have a ratio of 3:2, so we're going to have trouble selecting a widescreen ratio like in version 1 out of a portrait image. But I find it better balanced in version 2 anyway. Just try this with a portrait mode image.

And instead of the 4 apartments in a row, use 2, with bigger pics, as we wanted to use all over the site, where possible. The rest is fine.

**Comment by heliogabal** [comment]
you can use this one as an example of a portrait mode: 
https://herbert-kohlmeyer.de/sites/herbert-kohlmeyer.de/files/styles/referenz/public/referenzen/referenzerichweinertstr20ii.jpg

**Comment by darshi-shah** [comment]
Hi @heliogabal 

Please see attached mockup. Please review.

![object](https://cloud.githubusercontent.com/assets/15724372/25433795/50e49660-2aa7-11e7-827a-c7170bb079ac.png)


Thanks

**Comment by heliogabal** [comment]
yeah, could work. What about the bigger apartment pics like in #72 ? I'm curious if that would work to have only one in a row instead of two.

**Comment by saurabhd** [comment]
@neeraj-addweb please change the layout of apartments same as we did for the buy page and remove the blocks "Inquiry" and "contact person" from here.

**Comment by neeraj-addweb** [comment]
@heliogabal 
Updated design: 

![object](https://user-images.githubusercontent.com/18288247/31230711-70401fd6-aa02-11e7-962a-e2f4afb1f558.jpg)

**Comment by heliogabal** [comment]
That looks good, thank you. Just to make sure, let me confirm these few things: 

- Object should be the title of the project node
- Address should be only one size not two different sizes
- We don't need to provide any category here (refurbishment), as most of them will be the same. These categories are only relevant for the references page 
- The copy under the address is coming from the field field_project_house 
- There should be up to 10 units per page underneath
- The image should be the first from field_project_images and it should work for both portrait and landscape format (same width)

**Comment by neeraj-addweb** [comment]
@heliogabal 
Updated mockup as per your inputs.


![image](https://user-images.githubusercontent.com/18288247/31390375-586cc416-adf1-11e7-9805-1f73bdeec512.png)
